src/components/articles/viewArticleFooter.js

```diff
@@ -3,6 +3,7 @@ import * as PropTypes from 'prop-types';
 import styles from '../../styles/articleStyles/viewArticle.scss';
 import {USERNAME_KEY} from '../../constants/index';
```

Comment:
We do not have to import from `index`. You can remove the `/index`

Reply:
resolved